src/inference/src/dev/core_impl.cpp

```diff
@@ -1159,9 +1161,19 @@ ov::AnyMap ov::flatten_sub_properties(const std::string& device, const ov::AnyMa
     return result;
 }
 
+static void apply_moc_transformations(const std::shared_ptr<ov::Model>& model) {
+    ov::pass::Manager manager;
+    manager.register_pass<ov::pass::MOCTransformations>(false);
+    manager.register_pass<ngraph::pass::CompressQuantizeWeights>();
+    manager.register_pass<ngraph::pass::ZeroPointOptimizer>();
+    manager.run_passes(model);
+}
```

Comment:
Now we have a duplicate of this code in Pybind. Can we generalize this code please? Because they can become misaligned if someone forget to update some of these two functions.

Reply:
It's not really duplicate. Note that in pybind, CompressQuantizeWeights and MOCTransformations are in separate functions.